app/javascript/article-form/components/TagsField.jsx

```diff
@@ -0,0 +1,26 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import Tags from '../../shared/components/tags';
+
+export const TagsField = ({ defaultValue, onInput, switchHelpContext }) => {
+  return (
+    <div className="crayons-article-form__tagsfield">
+      <Tags
+        defaultValue={defaultValue}
+        maxTags="4"
+        onInput={onInput}
+        onFocus={switchHelpContext}
+        classPrefix="crayons-article-form"
+        fieldClassName="crayons-textfield crayons-textfield--ghost ff-accent fs-s s:fs-base"
```

Comment:
This is `14px` in narrow screen mode which [will cause zoom on iOS](https://stackoverflow.com/questions/2989263/disable-auto-zoom-in-input-text-tag-safari-on-iphone) which is a weird experience. Not sure about equivalent on Android.... But I think all inputs will need to be minimum 16px in all scenarios to work beautifully on touch devices.

Reply:
good catch, should be fixed.